File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/MPPFileUpdate.cs
using System.IO;
using Aspose.Tasks;
using System;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class MPPFileUpdate
    {
        public static void Run()
        {
            //ExStart: MPPFileUpdate
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();
            //Read an existing project
            Project project = new Project(dataDir+ "MPPFileUpdate.mpp");

            //create a new task
            Task task = project.RootTask.Children.Add("Task1");

            task.Set(Tsk.Start, new System.DateTime(2012, 8, 1));
            task.Set(Tsk.Finish, new DateTime(2012, 8, 5));

            //Save the project as MPP project file
            project.Save(dataDir+ "AfterLinking.Mpp", Aspose.Tasks.Saving.SaveFileFormat.MPP);
            //ExEnd: MPPFileUpdate   
        }
    }
}

File: Examples/VisualBasic/ManipulatingMPPFile/MPPFileUpdate.vb
Imports Aspose.Tasks
Imports System
Imports Aspose.Tasks.Saving

Namespace ManipulatingMPPFile
    Public Class MPPFileUpdate
        Public Shared Sub Run()
            'ExStart: MPPFileUpdate
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_ManipulatingMPPFile()
            'Read an existing project
            Dim project As New Project(dataDir & "MPPFileUpdate.mpp")

            'create a new task
            Dim task As Task = project.RootTask.Children.Add("Task1")

            task.Set(Tsk.Start, New System.DateTime(2012, 8, 1))
            task.Set(Tsk.Finish, New DateTime(2012, 8, 5))

            'Save the project as MPP project file
            project.Save(dataDir & "AfterLinking.Mpp", SaveFileFormat.MPP)
            'ExEnd: MPPFileUpdate
        End Sub
    End Class
End Namespace